CHANGELOG.md

```diff
@@ -0,0 +1,254 @@
+# Change Log
+
+## [v1.0.0](https://github.com/eclipse-iceoryx/iceoryx/tree/v1.0.0) (2020-04-01)
+
+[Full Changelog](https://github.com/eclipse-iceoryx/iceoryx/compare/v0.90.0...v1.0.0)
+
+Description:
+This is the first major release for Eclipse iceoryx. That means it is the first release with long-term support and the adopters of iceoryx can rely on a stable API. The release called Almond allows for true zero-copy inter-process-communication on Linux, QNX and MacOS and provides C and modern C++ user APIs.
```

Comment:
We should add the info here, how long the LTS is maintained aka "Supported until.."

FYI @budrus